Load the example app with a bigger size for have a open navigation pane and do not center the window

WinXaito opened the pull request:
I find it really annoying that the default Window size as a navigation pane is compact mode.

And each time a do a hot restart, I must change window size.

And at each hot restart, the window is replaced on the center. If we do not enable the center, with an hot restart, the window keep it's position.

---------------

@bdlukaa I know that you work on a laptop. Do you think these settings are good for you ? (For my part, I work on a 39" curved screen, so...)

--------------

Before:
![image](https://user-images.githubusercontent.com/8223773/228346426-3c798a91-6132-4375-a806-c6dcdf122769.png)

After:
![image](https://user-images.githubusercontent.com/8223773/228346469-574c73f0-52b2-4543-82bc-72dbd88fff80.png)


<!-- Add a description of what this PR is changing or adding, and why. Consider mentioning issues -->

## Pre-launch Checklist

<!-- Mark all that applies -->

- [x] I have run "dart format ." on the project <!-- REQUIRED -->

bdlukaa commented on example/lib/main.dart:
```diff
@@ -51,9 +51,8 @@ void main() async {
         TitleBarStyle.hidden,
         windowButtonVisibility: false,
       );
-      await windowManager.setSize(const Size(755, 545));
```
Maybe we don't need to make use of `.setSize`. We can set the window size from the windows runner. That is a better solution, that works at runtime and hot restart isn't going to break it

WinXaito commented:
Okay, I just removed the `setSize` (currently in windows\runner\main.cpp, the size is defined to 1280. That sounds good to me).

And I changed the min size:

Current:
![image](https://user-images.githubusercontent.com/8223773/228527690-879f4db0-c512-40ee-b03b-a48614169ae9.png)

New min size:
![image](https://user-images.githubusercontent.com/8223773/228527767-e3907ca1-386c-41b1-a598-61e6365cde54.png)
